web/src/unread.js

```diff
@@ -222,10 +217,64 @@ function make_per_stream_bucketer() {
     });
 }
 
+class UnreadTopicBucketer {
```

Comment:
I suspect that this commit isn't necessary -- one should be able to initialize a `Bucketer` with type parameters for the type of the keys and the type of the buckets, just like you can for a map, right?